diesel_tests/tests/bench.rs

```diff
@@ -163,7 +163,7 @@ bench_medium_complex_query!(
 
 #[bench]
 fn loading_associations_sequentially(b: &mut Bencher) {
-    // SETUP A FUCK TON OF DATA
+    // SETUP A TON OF DATA
```

Comment:
I felt the wording here should be changed after reading your https://github.com/diesel-rs/diesel/blob/master/code_of_conduct.md